google-cloud-pub-sub/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/Model.scala

```diff
@@ -4,10 +4,15 @@
 
 package akka.stream.alpakka.googlecloud.pubsub
 
+import java.time.Instant
+
 import scala.collection.immutable
 import scala.collection.JavaConverters._
 
-final case class PubSubMessage(data: String, messageId: String)
+final case class PubSubMessage(data: String,
+                               messageId: String,
+                               attributes: Option[immutable.Map[String, String]] = None,
+                               publishTime: Option[Instant] = None)
```

Comment:
Could you add a Java-friendly API with a backwards compatible contructor, `withAttributes(java.util.Map)` and `withPublishTime(Instant)` methods, please.